pipelines/api-pipelines.yml

```diff
@@ -86,12 +86,36 @@ stages:
               downloadType: 'single'
               artifactName: 'drop'
               downloadPath: '$(Build.ArtifactStagingDirectory)'
-             
+
+          - task: AzureCLI@1
+            displayName: Get Cosmosdb URI and Key
+            inputs:
+              azureSubscription: $(azureSubscription)
+              scriptLocation: inlineScript
+              inlineScript: |
+                set -eu
```

Comment:
nit: Extract this into a standalone script & reference this as scriptFile.  Much easier to read & debug.

Reply:
Sure. Will send out a separate PR for that as Jafar is dependent on this PR so I`m merging this one to unblock him.